### PR TITLE
Adding circleci yamls

### PR DESCRIPTION
**Pull request by KeenWill**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: 2.1
+orbs:
+  helm-tools: pennlabs/helm-tools@0.1.7
+
+workflows:
+  build-and-publish:
+    jobs:
+      - helm-tools/helm-publish:
+          chart-org: pennlabs
+          chart-repo: labscraft
```
Comment (pawalt) [inline .circleci/config.yml]: this needs to be `chart-repo: helm-charts`

**Review by pawalt — DISMISSED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (KeenWill) [inline .circleci/config.yml]:
```suggestion
          chart-repo: helm-charts
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+version: 2.1
+orbs:
+  helm-tools: pennlabs/helm-tools@0.1.7
```
Comment (ArmaanT) [inline .circleci/config.yml]: Bump to version 0.1.8 please

Comment (KeenWill) [inline .circleci/config.yml]:
```suggestion
  helm-tools: pennlabs/helm-tools@0.1.8
```

Comment (KeenWill) [reply .circleci/config.yml]: Update helm-tools version

**Review by ArmaanT — APPROVED** [review]

**Pull request merged**